datatypes/src/util/image.rs

```diff
@@ -16,6 +16,40 @@ pub fn assert_image_equals(expected: &Path, found: &[u8]) {
     assert_eq!(left, right, "Images differ: {expected:?}");
 }
 
+/// Compare two images
+///
+/// # Panics
+/// - if the `expected` image cannot be loaded
+/// - if the `found` bytes cannot be loaded as an image
+/// - if the images differ
+///
+pub fn assert_image_equals_with_format(expected: &Path, found: &[u8], format: ImageFormat) {
```

Comment:
so the `found` bytes do not contain the header or why is the `format` passed as an additional parameter?

Reply:
The crate wasn't able to auto detect it, but it works when I tell it that it is a Tiff.